x-pack/test/functional/apps/lens/group3/metric.ts

```diff
@@ -6,52 +6,18 @@
  */
 
 import expect from '@kbn/expect';
-import { WebElementWrapper } from '../../../../../../test/functional/services/lib/web_element_wrapper';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['visualize', 'lens', 'common', 'header']);
-  const findService = getService('find');
   const testSubjects = getService('testSubjects');
   const filterBar = getService('filterBar');
   const retry = getService('retry');
 
-  const getMetricTiles = () =>
```

Comment:
ℹ️ As we are using the new metric viz on the tests, I moved these functions to the lens page to be reusable.